### DISCUSSION TITLE
Not in full screen

### DISCUSSION THREAD

**Discussion by Foxx03** [GitSquared/edex-ui#1041] [Questions] [answered]
Hey, I have no idea what to do. edex should run in a full screen right? So, I have it installed on windows and about a week ago it dropped from fullscreen into borderless full sized window. So the task bar on the bottom of my screen is visible. How can I get it back to full screen.

**Comment by GitSquared** [comment, marked as answer]
Check your settings - ctrl+shift+s - you might have changed the `allowWindowed` or `forceFullscreen` options.

**Reply by Foxx03** [reply]
Thanks. I actually never noticed I can scroll for more settings. I had to manually add the forceFullscreen: true entry to settings.json. Now it’s fixed for me. Once again thanks.
